packages/board/test/redraw.test.js

```diff
@@ -185,21 +193,21 @@ describe('redraw', () => {
       expect(third.className).to.equal('medium');
     });
 
-    it('should only update class on resize or breakpoint change', () => {
+    it('should only update class on resize or breakpoint change', async () => {
       first.className = '';
-      board.redraw();
+      await aTimeout(100);
```

Comment:
Could we do anything to avoid this timeout?

Reply:
The test passes fine without the timeout also. It's there mainly there to avoid false positives (on current `master`, the test actually fails, if you add this timeout).

Reply:
Got it, thanks. I guess it's good to have it then.